dotcom-rendering/src/components/Picture.tsx

```diff
@@ -273,6 +273,29 @@ export const getFallbackSource = (sources: ImageSource[]): ImageSource => {
 	return fallback;
 };
 
+export const Sources = ({ sources }: { sources: ImageSource[] }) => {
```

Comment:
Not in the scope of this PR, but we could really do with a "generic" picture element 😅

Reply:
I was definitely thinking of this, I think it should be pretty easy to put together!